### ISSUE TITLE
Convert 647

### ISSUE THREAD

**Issue by kirkhess** [lcnetdev/marc2bibframe2#232]
While this is generally used for FAST headings and not used with LCSH, LC should have these in their catalog through CONSER and the structure is the same as a 648 FAST heading which is supported. 

Probably just adding '647' here will work.
https://github.com/lcnetdev/marc2bibframe2/blob/master/xsl/ConvSpec-600-662.xsl#L552C45-L552C45

**Comment by kirkhess** [comment]
Also, a Named Event implies a bf:Event but note there's no MADS/RDF equivalent; schema:Event is another option.

**Comment by kefo** [comment]
Can you point us to a sample record, please?  Preferably one that contains a FAST id.

**Comment by kirkhess** [comment]
There's about 7k CONSER records with 647. e.g. https://lccn.loc.gov/2023210267/marcxml

We're adding $1 to 647s to point to WC entities, eg. ocn 368128017

```
        <datafield tag="647" ind1=" " ind2="7">
          <subfield code="a">American Revolution</subfield>
          <subfield code="c">(United States :</subfield>
          <subfield code="d">1775-1783)</subfield>
          <subfield code="2">fast</subfield>
          <subfield code="0">(OCoLC)fst01351668</subfield>
          <subfield code="1">https://id.oclc.org/worldcat/entity/E39Qhp4vB989PWKg6VGKmH6MwK</subfield>
    </datafield>
```

In that case, the $1 is a [wco:Event](https://id.oclc.org/worldcat/ontology/Event), so something like this:

```
<bf:subject>
    <wco:Event rdf:about="https://id.oclc.org/worldcat/entity/E39Qhp4vB989PWKg6VGKmH6MwK">
        <rdfs:label>American Revolution (United States : 1775-1783)</rdfs:label>
        <bf:identifiedBy>
            <bf:Local>
                <rdf:value>fst01351668</rdf:value>
                <rdfs:label>(OCoLC)fst01351668</rdfs:label>
                <bf:source>
                    <bf:Source>
                        <rdf:value>OCoLC</rdf:value>
                    </bf:Source>
                </bf:source>
            </bf:Local>
        </bf:identifiedBy>
        <bf:source>
            <bf:Source bf:about="http://id.loc.gov/vocabulary/subjectSchemes/fast"/>
        </bf:source>
    </wco:Event>
</bf:subject>
```

**Comment by kirkhess** [comment]
wco:Event probably is a close match to bf:Event, although the proposed Event model is somewhat different (and untested...)

**Comment by jodiw01** [comment]
Included in marc2bibframe v2.5
